src/pages/Gift/Claim/index.tsx

```diff
@@ -12,7 +12,7 @@ export default function Claim({ classes = "" }) {
     },
     resolver: yupResolver(
       object({
-        secret: requiredString,
+        secret: requiredString.trim(),
```

Comment:
This whole page will probably be updated in the future, but updated its schema nonetheless.